app/controllers/static_pages_controller.rb

```diff
@@ -1,6 +1,10 @@
-
 class StaticPagesController < ApplicationController
-  def home; end
+  def home 
+    return if logged_in?
+    
+    @micropost = current_user.microposts.build
+    @feed_items = current_user.feed.paginate(page: params[:page]).per `number`
```

Comment:
```suggestion
    @feed_items = current_user.feed.paginate(page: params[:page]).per 123
```
move 123 to settings.yml file and call it here